src/main/kotlin/com/sourcegraph/cody/config/CodyApplicationSettings.kt

```diff
@@ -41,6 +42,8 @@ data class CodyApplicationSettings(
     this.isOnboardingGuidanceDismissed = state.isOnboardingGuidanceDismissed
     this.shouldAcceptNonTrustedCertificatesAutomatically =
         state.shouldAcceptNonTrustedCertificatesAutomatically
+    this.shouldCheckForUpdates = state.shouldCheckForUpdates
```

Comment:
was it a bug? 😅

Reply:
I think so :) Maybe the reason updater was not working? ;D